index.d.ts

```diff
@@ -7,6 +7,15 @@ declare namespace debounceFn {
 		*/
 		readonly wait?: number;
 
+		/**
+		Maximum time to wait until the `input` function is called.
```

Comment:
It needs a clearer description of how it's different from `wait`.

Reply:
It would also be useful to include a use-case for this option (like you mention in the PR description).

Reply:
You also need to update the readme.

Reply:
Also, make it clear that "time" is in milliseconds.

Reply:
I have tried to improve this. I'm not sure the first line is any better, but I don't how to make it better